packages/client/package.json

```diff
@@ -140,7 +140,7 @@
     }
   },
   "dependencies": {
-    "@prisma/engines-version": "4.2.0-10.a7370859a29d1da1fb77313857c210926d22396a"
+    "@prisma/engines-version": "4.2.0-2.query-engine-itx-isolation-17c351246f7ac88bb1914dfadfc973e2765366c4"
```

Comment:
Note before merging: use an engines version from main and not an integration one

Reply:
+1